.gitignore

```diff
@@ -1,3 +1,4 @@
 /node_modules
 package-lock.json
 apiary.html
+/.idea
```

Comment:
PP: not sure if we should add IDE specific ignores in here.

(feel free to ignore this)

Reply:
Here's a previous discussion on this;
https://bit.ly/2Izrffj

Reply:
We shouldn't indeed.

Reply:
I fixed that. Much better now.